packages/ckeditor5-emoji/src/emojipicker.ts

```diff
@@ -167,11 +167,20 @@ export default class EmojiPicker extends Plugin {
 		databaseId: number; title: string; exampleEmoji: string;
 	} ): Promise<EmojiGroup> {
 		const databaseGroup = await this._emojiDatabase.getEmojiByGroup( databaseId );
-
-		return {
-			title,
-			exampleEmoji,
-			items: databaseGroup.map( item => {
+		const container = this._createEmojiWidthTestingContainer();
+		const baselineEmojiWidth = this._getNodeWidth( container, '🙂' );
```

Comment:
I think that we do not need to calculate the baseline emoji width, but just store the container width as a baseline value:

```suggestion
		const baselineEmojiWidth = 24;
```

After that, it seems that all emojis are displayed correctly. Not supported emojis are also filtered out.

Reply:
Thanks, that looks good!